aztec/src/main/kotlin/org/wordpress/aztec/formatting/InlineFormatter.kt

```diff
@@ -40,6 +40,7 @@ class InlineFormatter(editor: AztecText, val codeStyle: CodeStyle, private val h
 
     data class CodeStyle(val codeBackground: Int, val codeBackgroundAlpha: Float, val codeColor: Int)
     data class HighlightStyle(@ColorRes val color: Int)
+    var markStyleColor: String? = null
```

Comment:
code styling nit pick: We could move this declaration below the `backgroundSpanColor` above